data/campaigns/Winds_of_Fate/scenarios/07_Wesmere.cfg

```diff
@@ -0,0 +1,638 @@
+        {UNIT 5 (Leopard) 22 5 ()}
+    [/side]
+
+    [side]
+        side=6
+        controller=ai
+        defeat_condition=no_units_left
+        recruit= Elvish Hero, Elvish Ranger, Elvish Marksman, Elvish Rider
+        {GOLD 150 300 450}
+        village_gold=5
+        team_name=elves
+        user_team_name= _ "Wesmere March Guard"
+        {FLAG_VARIANT wood-elvish}
+        [leader]
+            id=Vonel
+            name= _ "Vonel"
+            type=Elvish Marshal
+        [/leader]
+        [ai]
+            ai_algorithm=idle_ai
+        [/ai]
+        {NAMED_UNIT 6 (Elvish Rider) 20 23 (Nomeon) ( _ "Nomeon") ()}
+        # Guards inside Nomeon's border fort.
+        {UNIT 6 (Elvish Archer)  18 17 ()}
+        {UNIT 6 (Elvish Fighter) 18 20 ()}
+        {UNIT 6 (Elvish Fighter) 17 20 ()}
+        {UNIT 6 (Elvish Fighter) 15 19 ()}
+        {UNIT 6 (Elvish Fighter) 16 19 ()}
+        # Guards in watchtowers around the fort.
+        {UNIT 6 (Elvish Archer)  14 14 ()}
+        {UNIT 6 (Elvish Archer)  20 14 ()}
+        {UNIT 6 (Elvish Archer)  23 18 ()}
+        [village]
+            x,y=14,14
+        [/village]
+        [village]
+            x,y=20,14
+        [/village]
+        [village]
+            x,y=23,18
+        [/village]
+    [/side]
+
+    # wmllint: validate-on
+
+    [event]
+        name=prestart
+        {MODIFY_UNIT side=1 facing ne}
+        {MODIFY_UNIT side=2,3,4,6 facing sw}
+        [objectives]
+            [objective]
+                description= _ "Harvest all prey."
+                condition=win
+            [/objective]
+            [objective]
+                description= _ "Death of Gorlack"
+                condition=lose
+            [/objective]
+            [objective]
+                description= _ "Death of Reshan"
+                condition=lose
+            [/objective]
+            [objective]
+                description= _ "Death of Arinexis"
+                condition=lose
+            [/objective]
+            [objective]
+                description= _ "Death of Zedrix"
+                condition=lose
+            [/objective]
+            {TURNS_RUN_OUT}
+            [gold_carryover]
+                bonus=yes
+                carryover_percentage=20
+            [/gold_carryover]
+            [note]
+                description= _ "Our saurian 'adviser' Arinexis informs us this prey will remain calm until attacked."
+            [/note]
+            [note]
+                description= _ "This forest is poorly maintained, with an unusual amount of dry litter. Use of fire could ignite it.
+Creatures standing in burning forest will suffer some injury (16 fire damage)."
+            [/note]
+        [/objectives]
+    [/event]
+
+    [event]
+        name=start
+        [message]
+            speaker=Vonel
+            image_pos=right
+            mirror=yes
+            message= _ "You monsters there, Halt! This is the sovereign realm of Wesmere. No further may you go."
```

Comment:
I think that should be a full-stop before the 'Halt!'

Reply:
Or an exclamation mark.

Reply:
You monsters there! Halt!

or

You monsters there, halt!

Reply:
It's also a bit weird that they're talking to "monsters"…

Reply:
Makes sense, will probably go with: _"You monsters there! Halt!"_

His calling them monsters is derogatory, since the elves should know that at least the saurians are capable of speech and complex thought.